src/main/scala/rocket/RocketCore.scala

```diff
@@ -39,7 +39,7 @@ case class RocketCoreParams(
   fastLoadWord: Boolean = true,
   fastLoadByte: Boolean = false,
   branchPredictionModeCSR: Boolean = false,
-  clockGate: Boolean = false,
+  //clockGate: Boolean = false,
```

Comment:
But this is still something that RocketCores can configure right? Why are you remove the ability to configure it?

Reply:
Yes, RocketCores can configure it. I just move this clockGate from rocketParams to CoreParams, so any module extends HasCoreParams can see this value.